src/components/Collection/CleanDescription.tsx

```diff
@@ -4,23 +4,23 @@ import cx from 'classnames';
 import { useSettingsQuery } from '@/core/react-query/settings/queries';
 
 // The question marks are there because people can't spell…
-const RemoveSummaryRegex = /\b(Sour?ce|Note|Summ?ary):([^\r\n]+|$)/mg;
+const CleanInfoLinesRegex = /\b((Modified )?Sour?ce|Note( [1-9])?|Summ?ary):(?!$)([^\r\n]+|$)/img;
```

Comment:
only question i have is why is the negative lookahead for the end of text needed?

Reply:
For something like this: https://anidb.net/anime/9283